src/pg/brc20/brc20-pg-store.ts

```diff
@@ -273,6 +321,42 @@ export class Brc20PgStore extends BasePgStoreModule {
       );
   }
 
+  async rollBackInscription(args: { inscription: DbInscription }): Promise<void> {
+    const activities = await this.sql<DbBrc20Activity[]>`
```

Comment:
Might want a different type here than `DbBrc20Activity`, if we're not joining with anything.

Reply:
Added in PR #215